ci-operator/step-registry/rosa/aws/sts/imdsv2/rosa-aws-sts-imdsv2-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: rosa-aws-sts-imdsv2
+  steps:
+    env:
+      COMPUTE_MACHINE_TYPE: "m5.2xlarge"
+      MULTI_AZ: "true"
+      ENABLE_AUTOSCALING: "true"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
+      EC2_METADATA_HTTP_TOKENS: "required"
```

Comment:
This is the [enhancement doc](https://github.com/openshift/enhancements/blob/master/enhancements/machine-api/aws-imds-v2-support.md#summary), you can take a look, it can increase the security of the instances. I think it can be used as a basic workflow, in future if we need to add other workflows we can override it. But I think we shouldn't override the default rosa sts cluster, in most cases, the http-token is optional.